source/adios2/toolkit/sst/CMakeLists.txt

```diff
@@ -45,8 +45,7 @@ if(ADIOS2_HAVE_ZFP)
   target_link_libraries(sst PRIVATE zfp::zfp)
 endif()
 
-if(ADIOS2_SST_HAVE_MPI_CLIENT_SERVER)
-  set(SST_HAVE_MPI_DP TRUE)
+if(ADIOS2_SST_BUILD_MPI_DP)
```

Comment:
Does this work for you?  I don't see ADIOS2_SST_BUILD_MPI_DP set anywhere and those files don't get compiled for me.